src/resolvers/user.js

```diff
@@ -0,0 +1,12 @@
+import { User } from '../models'
+
+export default {
+  Mutation: {
+    async createUser(root, { data: { name, type } }) {
```

Comment:
We should prefix mutation names with the object they are mutating for alphabetical grouping (in graphiql) e.g. use `userCreate` instead of `createUser`.

I know its just a starter but lets start with good practices from the beginning...